libavpipe/include/avpipe_xc.h

```diff
@@ -112,7 +112,7 @@ typedef struct udp_packet_t {
 typedef struct mux_input_ctx_t {
     int     n_parts;                    /* Number of input parts */
     int     index;                      /* Index of current input part that should be processed */
-    char    *parts[MAX_MUX_IN_STREAM];  /* All the input parts */
+    char    **parts;                    /* All the input parts */
```

Comment:
Is this better than the array?  Sine we are allocating the entire amount anyway?